notebooks/03-query/17-table-append-operations.ipynb

```diff
@@ -0,0 +1,479 @@
+    "SELECT\n",
+    "  TIME_PARSE(\"timestamp\") AS \"__time\",\n",
+    "  'table2' AS tableName,\n",
+    "  \"isRobot\",\n",
+    "  \"channel\",\n",
+    "  \"page\",\n",
+    "  \"commentLength\",\n",
+    "  \"countryName\",\n",
+    "  \"user\"\n",
+    "FROM \"ext\"\n",
+    "WHERE \"isRobot\"=FALSE\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "\n",
+    "display.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-wikipedia-table-append2')\n",
+    "\n",
+    "sql = '''\n",
+    "SELECT *\n",
+    "FROM \"example-wikipedia-table-append2\"\n",
+    "WHERE TIME_IN_INTERVAL(__time, '2016-06-27/2016-06-28')\n",
+    "ORDER BY __time LIMIT 5\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24b47cc3-68f5-4a73-b374-94bbfa32d91d",
+   "metadata": {},
+   "source": [
+    "You can now use TABLE(APPEND) to append one table to another.\n",
+    "\n",
+    "The first query in the cell below returns the ten first edits in the resulting table which combines both the existing tables. You can see which table each record is orginally from by looking at the `tableName` field."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b76e5184-9fe4-4f21-a471-4e15d16515c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = '''\n",
+    "SELECT *\n",
+    "FROM TABLE(APPEND('example-wikipedia-table-append1','example-wikipedia-table-append2'))\n",
+    "WHERE TIME_IN_INTERVAL(__time, '2016-06-27/2016-06-28')\n",
+    "ORDER BY __time LIMIT 10\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5e77fa9-a60c-4955-b763-58d970d7326d",
+   "metadata": {},
+   "source": [
+    "Optionally, run the next cell to show the precise [`EXPLAIN PLAN`](https://druid.apache.org/docs/latest/querying/sql-translation#interpreting-explain-plan-output) for the query. There is one `query` execution plan (along with Druid's planning process optimized execution of the outer query.), as opposed to the two for a `UNION` operation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97934da2-17d1-4c91-8ae3-926cc89185c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(json.dumps(json.loads(sql_client.explain_sql(sql)['PLAN']), indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f23a2e8-8a31-4ae0-8c95-fa071de6afe8",
+   "metadata": {},
+   "source": [
+    "Run the below to get the total number of edits made by humans and the total number of edits made by bots. Without TABLE(APPEND) it would be necessary\n",
```

Comment:
it would be necessary to ? i think there is text missing.

also note:
'you can' is more concise than 'it is possible to'